examples/CustomAssignmentConsoleApp/appsettings.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "FeatureManagement": {
+    "DailyDiscount": {
+      "Assigner": "Recurring",
+      "Variants": [
+        {
+          "Default": true,
+          "Name": "Default",
+          "ConfigurationReference": "DailyDiscount:Default"
+        },
+        {
+          "Name": "Tuesday",
+          "ConfigurationReference": "DailyDiscount:Tuesday",
+          "AssignmentParameters": {
+            "Days": [
+              "Tuesday"
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "DailyDiscount": {
+    "Default": {
+      "Discount": 20,
+      "ProductName": "Bananas"
+    },
+    "Tuesday": {
```

Comment:
>Tuesday [](http://example.com/codeflow?start=5&length=7)

Nice sample. The configuration is independent from the assignment. Maybe name this "Special"? It can be assigned on any day.